misaligned/misaligned.cs

```diff
@@ -3,21 +3,33 @@
 
 namespace MisalignedSpace {
     class Misaligned {
-        static int printColorMap() {
-            string[] majorColors = {"White", "Red", "Black", "Yellow", "Violet"};
-            string[] minorColors = {"Blue", "Orange", "Green", "Brown", "Slate"};
-            int i = 0, j = 0;
-            for(i = 0; i < 5; i++) {
-                for(j = 0; j < 5; j++) {
+        static int GetTotalCount(string[] majorColors, string[] minorColors)
+        {
+            return majorColors.Length * minorColors.Length;
+        }
+
+        static void PrintColorMap(string[] majorColors, string[] minorColors)
+        {
+            for (int i = 0; i < majorColors.Length; i++)
+            {
+                for (int j = 0; j < minorColors.Length; j++)
+                {
                     Console.WriteLine("{0} | {1} | {2}", i * 5 + j, majorColors[i], minorColors[i]);
                 }
             }
-            return i * j;
         }
+
         static void Main(string[] args) {
-            int result = printColorMap();
-            Debug.Assert(result == 25);
+            string[] majorColors = { "White", "Red", "Black", "Yellow", "Violet" };
+            string[] minorColors = { "Blue", "Orange", "Green", "Brown", "Slate" };
+
+            PrintColorMap(majorColors, minorColors);
+
+            Debug.Assert(GetTotalCount(majorColors, minorColors) != 25);

```

Comment:
Have the assert express some functionality. This assert doesn't seem to be related to the reference-manual functionality.